release.json

```diff
@@ -29,28 +29,28 @@
         "SECURITY_AGENT_POLICIES_VERSION": "master"
     },
     "release-a6": {
-        "INTEGRATIONS_CORE_VERSION": "7.38.0-rc.3",
-        "OMNIBUS_SOFTWARE_VERSION": "7.38.0-rc.1",
-        "OMNIBUS_RUBY_VERSION": "7.38.0-rc.1",
+        "INTEGRATIONS_CORE_VERSION": "7.39.0-rc.2",
```

Comment:
Is it expected to have rc2?

Reply:
yes, it was confirmed with Agent-Integrations team